docs/src/main/tut/docs/http4sExamples/auth-jwt.md

```diff
@@ -32,43 +32,39 @@ Notes:
 ### Authenticator Creation
 
 ```tut:silent
-  import java.util.UUID
-  import cats.effect.IO
-  import examples.Http4sAuthExample._
-  import examples.Http4sAuthExample.User._
-  import examples.Http4sAuthExample.Role._
-  import tsec.authentication._
-  import tsec.authorization._
-  import tsec.mac.imports._
-  import org.http4s.HttpService
-  import org.http4s.dsl.io._
-  import scala.concurrent.duration._
-  import tsec.common._
-  import tsec.jws.mac.JWTMac
-  import tsec.cipher.symmetric._
-  import tsec.cipher.symmetric.imports._  
+import cats.effect.IO
+
+import org.http4s.HttpService
+import org.http4s.dsl.io._
+import tsec.authentication._
+import tsec.common.SecureRandomId
+import tsec.mac.imports.{HMACSHA256, MacSigningKey}
+import scala.concurrent.duration._
+
+object jwtStatelessExample {
+
+  import http4sExamples.ExampleAuthHelpers._
 
   val jwtStore =
-    dummyBackingStore[IO, SecureRandomId, JWTMac[HMACSHA256]](s => SecureRandomId.coerce(s.id))
+    dummyBackingStore[IO, SecureRandomId, AugmentedJWT[HMACSHA256, Int]](s => SecureRandomId.coerce(s.id))
 
   //We create a way to store our users. You can attach this to say, your doobie accessor
   val userStore: BackingStore[IO, Int, User] = dummyBackingStore[IO, Int, User](_.id)
 
   val settings: TSecJWTSettings = TSecJWTSettings(
-    expirationTime = 10.minutes, //Absolute expiration time
+    expiryDuration = 10.minutes, //Absolute expiration time
     maxIdle = None
   )
 
-  val signingKey: MacSigningKey[HMACSHA256] = HMACSHA256.generateKeyUnsafe() //Our signing key. Instantiate in a safe way using GenerateLift
-  val encryptionKey: SecretKey[AES128] = AES128.generateKeyUnsafe() //Our encryption key. Instantiate in a safe way using GenerateLift
+  val signingKey
+  : MacSigningKey[HMACSHA256] = HMACSHA256.generateKeyUnsafe() //Our signing key. Instantiate in a safe way using GenerateLift
 
   val jwtStatelessauth =
-    JWTAuthenticator.withBackingStore(
+    JWTAuthenticator.withBackingStoreArbitrary(
```

Comment:
no mention of using an arbitrary header for transport.

Defaulting to `Authenticator: Bearer` is a choice I made during the PR for M5. This is the only thing I'd change about your thing.